src/client/App.jsx

```diff
@@ -15,15 +15,76 @@ function App() {
   }, []);
 
   const handleRegister = async ({ username, password }) => {
-    
+
+    const options = {
+      method: 'POST',
+      headers: {
+          'content-type': 'application/json',
+      },
+      body: JSON.stringify({
+          username: username,
+          password: password
+      }),
+  }
+
+  fetch('http://localhost:4000/user/register', options)
+      .then(res => res.json())
+      .then(json => console.log('Registered User:' + json.data.username))
+      .catch(() => {
+          console.log('server error')
+      })
   };
 
   const handleLogin = async ({ username, password }) => {
     
+    const options = {
+      method: 'POST',
+      headers: {
+          'content-type': 'application/json',
+      },
+      body: JSON.stringify({
+          username: username,
+          password: password
+      }),
+  }
+
+  fetch('http://localhost:4000/user/login', options)
+      .then(res => res.json())
+      .then(json => {
+        console.log('logged in:' + json.data)
+        localStorage.setItem('jwt', json.data)
+      })
   };
   
   const handleCreateMovie = async ({ title, description, runtimeMins }) => {
-    
+
+    const options = {
+      method: 'POST',
+      headers: {
+          'content-type': 'application/json',
+          'authorization': 'Bearer ' + localStorage.getItem('jwt')
+      },
+      body: JSON.stringify({
+         title: title,
+         description: description,
+         runtimeMins: runtimeMins
+      }),
+  }
+
+  fetch('http://localhost:4000/movie', options)
+  .then(res => {
+    res.json().then(json => {
+      if(res.ok) {
```

Comment:
I didn't know about Response.ok, just looked it up: https://developer.mozilla.org/en-US/docs/Web/API/Response/ok, this is quite handy!